### PR TITLE
WT-7269 Add column store to existing RTS tests

### PR DESCRIPTION
**Pull request by kommiharibabu**
A couple of fixes along with added tests:
- Mark the page dirty whenever RTS aborted updates on the page to let the reconcile to write a stable update to disk
- Pass the restored update as a base update for column store search instead of on-disk value

### CODE REVIEW

**Comment by kommiharibabu** [comment]
@tammybailey  and @raviprakashgiri29 can you review this PR change of adding column store support to existing tests and fixes.

File: test/suite/test_rollback_to_stable16.py [hunk 1]
```diff
@@ -30,27 +30,62 @@
...
         cursor.close()
 
     def check(self, check_value, uri, nrows, start_row, read_ts):
         session = self.session
-        session.begin_transaction('read_timestamp=' + timestamp_str(read_ts))
+        if read_ts == 0:
+            session.begin_transaction()
+        else:
+            session.begin_transaction('read_timestamp=' + timestamp_str(read_ts))
+        # session.begin_transaction('read_timestamp=' + timestamp_str(read_ts))
```
Comment (tammybailey) [inline test/suite/test_rollback_to_stable16.py:88]: Should this line be removed?

File: test/suite/test_rollback_to_stable16.py [hunk 2]
```diff
@@ -59,10 +94,14 @@ def check(self, check_value, uri, nrows, start_row, read_ts):
             ret = cursor.search()
             if check_value is None:
                 if ret != wiredtiger.WT_NOTFOUND:
-                    self.tty(f'value = {cursor.get_value()}')
+                    # self.tty(f'key = {cursor.get_key()}')
```
Comment (tammybailey) [inline test/suite/test_rollback_to_stable16.py:97]: Should these lines be removed?

File: test/suite/test_rollback_to_stable16.py [hunk 3]
```diff
@@ -93,16 +135,27 @@ def test_rollback_to_stable16(self):
...
-
-        # Rollback to stable done as part of recovery.
-        simulate_crash_restart(self,".", "RESTART")
+        if not self.in_memory:
+            # Checkpoint to ensure that all the updates are flushed to disk.
+            self.session.checkpoint()
+            # Rollback to stable done as part of recovery.
+            simulate_crash_restart(self,".", "RESTART")
+        else:
+            # Manually call rollback_to_stable for in memory k/v's.
```
Comment (tammybailey) [inline test/suite/test_rollback_to_stable16.py:144]: No apostrophe in keys/values.

File: test/suite/test_rollback_to_stable17.py [hunk 4]
```diff
@@ -86,11 +106,14 @@ def test_rollback_to_stable(self):
...
-
-        # Rollback to stable done as part of recovery.
-        simulate_crash_restart(self, ".", "RESTART")
+        if not self.in_memory:
+            # Checkpoint to ensure that all the updates are flushed to disk.
+            self.session.checkpoint()
+            # Rollback to stable done as part of recovery.
+            simulate_crash_restart(self,".", "RESTART")
+        else:
+            # Manually call rollback_to_stable for in memory k/v's.
```
Comment (tammybailey) [inline test/suite/test_rollback_to_stable17.py:115]: See above comment!

**Review by tammybailey — APPROVED** [review]
Just a few small comments on comments, LGTM otherwise.

File: test/suite/test_rollback_to_stable16.py (hunk 1 above)
Comment (kommiharibabu) [reply test/suite/test_rollback_to_stable16.py]: Removed.

File: test/suite/test_rollback_to_stable16.py (hunk 2 above)
Comment (kommiharibabu) [reply test/suite/test_rollback_to_stable16.py]: Removed.

File: test/suite/test_rollback_to_stable16.py (hunk 3 above)
Comment (kommiharibabu) [reply test/suite/test_rollback_to_stable16.py]: Fixed as suggested.

File: test/suite/test_rollback_to_stable17.py (hunk 4 above)
Comment (kommiharibabu) [reply test/suite/test_rollback_to_stable17.py]: Fixed as suggested.

**Review by raviprakashgiri29 — APPROVED** [review]
Changes LGTM

**Pull request merged**